docs/static/releasenotes.asciidoc

```diff
@@ -70,28 +70,18 @@ chain TLS certificate verification.
 [[plugins-7-15-2]]
 ==== Plugins
 
-*Geoip Filter - 7.2.4*
-
-* Fix: update to Gradle 7 https://github.com/logstash-plugins/logstash-filter-geoip/pull/191[#191]
-* [DOC] Clarify CC licensed database indefinite use condition and air-gapped environment https://github.com/logstash-plugins/logstash-filter-geoip/pull/192[#192]
```

Comment:
Is the 7.2.4 update coming out for a reason? If it's just because these are "minor changes," I'd suggest we leave them in. If a user in really interested in a particular plugin, we should give them the info and let them decide for themselves if they're interested or not.